### DISCUSSION TITLE
Prebuilt for 2.32.4for Debian Buster amd64?

### DISCUSSION THREAD

**Discussion by dma9** [trapexit/mergerfs#882] [Q&A] [answered]
Please excuse the dumb question, but I can't seem to find the above on https://github.com/trapexit/mergerfs/releases. All I see is the one for armhf. Is there somewhere else I might find it? Not a huge deal as I can build from source but just thought I'd take the liberty of asking before doing so.

**Comment by trapexit** [comment]
There has been something odd with the build images I use and some aren't building. Particularly armhf. Let me try kicking off a build again to see. I mentioned this in the 2.32.3 release but forgot in .4.

**Reply by trapexit** [reply]
Updated. Sorry about that.

**Reply by dma9** [reply]
Oh hey no worries at all. I'm just grateful to have the opportunity to use such well-crafted code. 

And thank you very much for updating the releases. Most appreciated.

**Comment by trapexit** [comment, marked as answer]
Fixed: https://github.com/trapexit/mergerfs/releases/tag/2.32.4
